Makefile

```diff
@@ -78,20 +78,15 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-CONTAINER_RUNTIME ?= docker
 
 CURRENT_CONTEXT := $(shell [[ "`command -v kubectl`" != '' ]] && kubectl config current-context 2> /dev/null || echo "unset")
 
-
 # CONTAINER_TOOL defines the container tool to be used for building images.
 # Be aware that the target commands are only tested with Docker which is
 # scaffolded by default. However, you might want to replace it to use other
 # tools. (i.e. podman)
 CONTAINER_TOOL ?= docker
-CONTAINER_TOOL:=$(shell command -v docker 2> /dev/null)
-ifndef CONTAINER_TOOL
-CONTAINER_TOOL:=$(shell command -v podman 2> /dev/null)
-endif
+
```

Comment:
Since some of us are aliasing "docker" to "podman", I didn't want to leave this in, because it will cause `CONTAINER_TOOL=docker`, and that will perform the wrong logic in `make image-import`.